### PR TITLE
Documentation for New/Get/Remove-CsHybridTelephoneNumber

### PR DESCRIPTION
**Pull request by JensTrier**
Adding documentation for GCCH and DOD only cmdlets.

### CODE REVIEW

File: teams/teams-ps/teams/Get-CsHybridTelephoneNumber.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+### Assignment (Default)
+```powershell
+Get-CsHybridTelephoneNumber [-TelephoneNumber <string>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+This cmdlet displays information about one or more hybrid telephone numbers used for Audio Conferencing with Direct Routing for GCC High and DoD clouds.
+
+Returned results are sorted by TelephoneNumber in ascending order.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsHybridTelephoneNumber.md:28]:
```suggestion
Returned results are sorted by telephone number in ascending order.

```

File: teams/teams-ps/teams/Get-CsHybridTelephoneNumber.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+```powershell
+Get-CsHybridTelephoneNumber
+```
+```output
+Id              O365Region SourceType TargetType TelephoneNumber UserId
+--              ---------- ---------- ---------- --------------- ------
+14025551234                Hybrid                14025551234
+14025551235                Hybrid                14025551235
+```
+This example displays information about all hybrid telephone phone numbers in the tenant. Note that O365Region, TargetType and UserId will not be populated.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsHybridTelephoneNumber.md:53]:
```suggestion
This example displays information about all hybrid telephone numbers in the tenant. Note that O365Region, TargetType, and UserId will not be populated.

```

File: teams/teams-ps/teams/Get-CsHybridTelephoneNumber.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+## PARAMETERS
+
+### -TelephoneNumber
+Filters the returned results to a specific phone number. The number should be specified without a prefixed "+". The phone number can not have "tel:" prefixed.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsHybridTelephoneNumber.md:59]:
```suggestion
Filters the returned results to a specific phone number. The number should be specified without a prefixed "+". The phone number cannot have "tel:" prefixed.

```

File: teams/teams-ps/teams/New-CsHybridTelephoneNumber.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+This cmdlet adds a hybrid telephone number to the tenant that can be used for Audio Conferencing with Direct Routing for GCC High and DoD clouds.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+New-CsHybridTelephoneNumber -TelephoneNumber +14025551234
+```
+This example adds the hybrid phone number +1 (402) 555-1234.
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsHybridTelephoneNumber.md:35]:
```suggestion

```

File: teams/teams-ps/teams/Remove-CsHybridTelephoneNumber.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+This cmdlet removes a hybrid telephone number used for Audio Conferencing with Direct Routing for GCC High and DoD clouds.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+Remove-CsHybridTelephoneNumber -TelephoneNumber 14025551234
+```
+This example removes the hybrid phone number +1 (402) 555-1234.
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-CsHybridTelephoneNumber.md:35]:
```suggestion

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

File: teams/teams-ps/teams/Get-CsHybridTelephoneNumber.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+Get-CsHybridTelephoneNumber
+```
+```output
+Id              O365Region SourceType TargetType TelephoneNumber UserId
+--              ---------- ---------- ---------- --------------- ------
+14025551234                Hybrid                14025551234
+14025551235                Hybrid                14025551235
+```
+This example displays information about all hybrid telephone phone numbers in the tenant. Note that O365Region, TargetType and UserId will not be populated.
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsHybridTelephoneNumber.md:54]:
```suggestion

```

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

Thanks @JohanFreelancer9 & done!

File: teams/teams-ps/teams/Get-CsHybridTelephoneNumber.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+### Assignment (Default)
+```powershell
+Get-CsHybridTelephoneNumber [-TelephoneNumber <string>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+This cmdlet displays information about one or more hybrid telephone numbers used for Audio Conferencing with Direct Routing for GCC High and DoD clouds.
+
+Returned results are sorted by telephone number in ascending order.
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsHybridTelephoneNumber.md:29]:
```suggestion

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @JensTrier, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips @tseward

**Review by tseward — APPROVED** [review]

**Pull request merged**